docs/guide/built-in-rules-callback.md

```diff
@@ -1,20 +1,20 @@
-# `Callback` - a wrapper over `callable`
+# `Callback` - a wrapper around `callable`
 
-This rule allows validation of current attribute value (but not limited to it) with an arbitrary condition set within a 
-callable. The benefit of it is that creation of a separate custom rule and handler is not required.
+This rule allows you to validate the current attribute value (but not limited to it) with an arbitrary condition within a  
```

Comment:
What's the reason of this change (except the article)?

Reply:
`it` was replaced to `this type of validator` because it is not clear what it was referring to: rule or attribute value. I'll replace "this type of validator" with "this type of rule".